blinkt.go

```diff
@@ -93,17 +119,39 @@ func (bl *Blinkt) Show() {
 }
 
 // SetAll sets all pixels to specified r, g, b colour. Show must be called to update the LEDs.
-func (bl *Blinkt) SetAll(r int, g int, b int) {
+func (bl *Blinkt) SetAll(r int, g int, b int, brightness ...float64) {
+
 	for i, _ := range bl.pixels {
 		bl.SetPixel(i, r, g, b)
 	}
+
+	if len(brightness) > 0 {
+		bl.SetBrightness(brightness[0])
```

Comment:
Without meaning to sound flippant, the second and any n>1 values will be ignored.

I think this goes away with method chaining.